dev-client/src/screens/HomeScreen/components/TemporaryLocationCallout.tsx

```diff
@@ -52,16 +52,23 @@ export const TemporaryLocationCallout = ({
   const {t} = useTranslation();
   const navigation = useNavigation();
   const [siteElevationString, setSiteElevationString] = useState('');
+  const [siteElevationValue, setSiteElevationValue] = useState(0);
```

Comment:
Suggestion: Can we remove the siteElevationString state stuff now? Like, just do `useState` for the elevation value, and then derive the elevation string from the value when we need the string? It seems less clean to save both as separate state variables when one is easily derived from the other.

Reply:
Good catch.